packages/web-api-scan-job-manager/src/batch/batch-config.spec.ts

```diff
@@ -0,0 +1,33 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+import 'reflect-metadata';
+
+import { BatchConfig } from './batch-config';
+
+describe(BatchConfig, () => {
+    let batchConfig: BatchConfig;
+
+    it('return value of AZ_BATCH_ACCOUNT_NAME environment variable', () => {
+        const value: string = `value-${new Date().valueOf()}`;
```

Comment:
>`value-${new Date().valueOf()} [](start = 30, length = 30)

this could be a constant value.
also you could just have 1 unit test that validates all 3 properties.:
process.env.AZ_BATCH_ACCOUNT_NAME = "test batch account1";
process.env.AZ_BATCH_ACCOUNT_URL = "test account url1";
...

expect(batchConfig.accountName).toEqual(process.env.AZ_BATCH_ACCOUNT_NAME);
...

Reply:
this is the same in the old package, I'd like to focus on what was changed first for the sake of time